certified-connectors/Zellis/apiProperties.json

```diff
@@ -0,0 +1,300 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "scopes": [
+            "{Scope}"
+          ],
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```

Comment:
First of all I do not understand why we used GlobalPerConnector as we ealier used Global only but @vmanoharas told us to use GlobalPerConnector. However your below certified connector using Global only.
https://github.com/microsoft/PowerPlatformConnectors/blob/60f4083b0e817021d7912d12e1b484027b4655a9/certified-connectors/CardsForPowerApps/apiProperties.json#L13

And why we need to change the redirectUrl as its working for us. There is no proper documentation about anything and now since our connector is working with lot of hard work you guys just suggesting random changes. This is not acceptable.

Reply:
Hi @avgode-zellis, 

I apologize for any prior confusion. As mentioned earlier, the update from "Global" to "GlobalPerConnector" is indeed correct. Additionally, the URL requested by @SaiChander-MS  should also be updated to align with recent process changes. It's important to note that this change is mandatory.

Although the documentation hasn't been updated as of yet, we will share the revised version once it is ready. We appreciate your patience and understanding.

Reply:
@SaiChander-MS @vmanoharas If you not updated any documentation how we suppose to update it. And there is testing needs to be done who will compensate for that? Also where I suppose to add that URL and how... Any example, since we do not have any API as such.

Reply:
> @SaiChander-MS @vmanoharas If you not updated any documentation how we suppose to update it. And there is testing needs to be done who will compensate for that? Also where I suppose to add that URL and how... Any example, since we do not have any API as such.

Hello @avgode-zellis, I completely understand your concerns, and I want to assure you that we're here to assist you through this process. Updating the documentation can sometimes be challenging. Kindly ignore updating the URL as of now, our engineering team will take care of this and guide you regarding testing when it is uploaded into the tool.